### PR TITLE
Fixes #19113 - use fog-ovirt method for rebooting

### PR DESCRIPTION
**Pull request by iNecas**
While fixing the waiting for stopped vm in reboot there.

Waiting for https://github.com/fog/fog/pull/3954 to get in.

### CODE REVIEW

File: app/models/concerns/fog_extensions/ovirt/server.rb
Comment (tbrisker) [inline app/models/concerns/fog_extensions/ovirt/server.rb:37]: :+1: to pushing this into fog.

**Review by tbrisker — APPROVED** [review]
Looks good, will merge once fog change is merged and released.

**Review by dLobatog — CHANGES_REQUESTED** [review]
@iNecas 👍 - I would say 1.16, as I'm not too keen on updating the fog version for a patch version... Setting to WoC so you can let us know when the release has been done.

**Comment by iNecas** [comment]
@dLobatog @tbrisker new fog, that includes required change got just merged. The required version is bumped and ready for final re-review

**Review by dLobatog — APPROVED** [review]

**Review by dLobatog — CHANGES_REQUESTED** [review]
@iNecas I think you also need to update the fog-core dep, after that I'm 👍  if tests pass.

```
Bundler could not find compatible versions for gem "fog-core":
  In Gemfile:
    fog-core (= 1.44.2)

    fog (= 1.41.0) was resolved to 1.41.0, which depends on
      fog-core (~> 1.45)
```

**Comment by iNecas** [comment]
Updated the deps, waiting for the tests

**Comment by iNecas** [comment]
[test foreman]

**Comment by iNecas** [comment]
[test foreman]

**Review by dLobatog — CHANGES_REQUESTED** [review]
@iNecas Seems that tests fail?

**Comment by iNecas** [comment]
[test foreman]

Couldn't reproduce locally still :( Hoping it is something with Jenkins :)

**Comment by iNecas** [comment]
Tests are 🍏 now

**Review by dLobatog — APPROVED** [review]

**Comment by dLobatog** [comment]
Thanks @iNecas !

**Pull request merged**
